README.md

```diff
@@ -480,6 +492,7 @@ h
 * [Google Web Toolkit](http://www.gwtproject.org/) - Toolbox which includes a Java-to-JavaScript compiler for client-side code, XML parser, API for RPC, JUnit integration, internationalization support and widgets for the GUI.
 * [Grails](https://grails.org/) - Groovy framework with the aim to provide a highly productive environment by favoring convention over configuration, no XML and support for mixins.
 * [Ninja](http://www.ninjaframework.org/) - Full stack web framework for Java. Rock solid, fast and super productive.
+* [Oracle ADF](http://www.oracle.com/technetwork/developer-tools/adf/overview/index.html) - Providers a free and a commercial version framework to write web and mobile applications, following an event-driven approach.
```

Comment:
This list will never include the ADF framework. It is the exact opposite of the definition awesome lists have.